hypex/algorithms/faiss_matcher.py

```diff
@@ -121,7 +118,8 @@ def __init__(
         if group_col is None:
             self.df = df
         else:
-            self.df = df.sort_values([treatment, group_col])
+            group_col = group_col if isinstance(group_col, list) else [group_col]
+            self.df = df.sort_values([treatment, group_col[0]])
```

Comment:
I'm not sure that it would work correctly. May be it better should do after preprocessing

Reply:
Check matcher.py, actually it's ok. May be would be better set group_col here for str after preprocessing, but for now it's ok. Will change in dev